### PR TITLE
Add 3 missed.

### PR DESCRIPTION
**Pull request by ROCO-freedom**
When we made the Stable language patch, we found that several .nut files were updated, so we added the corresponding .map files for them.

### CODE REVIEW

File: base_tasofro/th175/data/script/scene/pause.v1.14.nut.map [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	"/Functions/2/Literals/66": "th175_09",
+	"/Functions/2/Literals/68": "th175_10",
+	"/Functions/2/Literals/70": "th175_11",
+	"/Functions/2/Literals/72": "th175_12",
+	"/Functions/2/Literals/74": "th175_13",
+	"/Functions/2/Literals/76": "th175_14",
+	"/Functions/2/Literals/78": "th175_15",
+	"/Functions/2/Literals/80": "th175_16",
+	"/Functions/2/Literals/82": "th175_17",
+	"/Functions/2/Literals/84": "th175_18_1",
```
Comment (brliron) [inline base_tasofro/th175/data/script/scene/pause.v1.14.nut.map:19]: Why th175_18_1? I think these IDs should be the same as in the themes database (https://www.thpatch.net/wiki/Touhou_Patch_Center:List_of_music_themes). Same for th175_24_1 below.

Comment (ROCO-freedom) [reply base_tasofro/th175/data/script/scene/pause.v1.14.nut.map:19]: When I was looking for pause.nut, the names of these two BGMs did not have "\S[XX]", and these IDs were already taken, so I added two new strings and named their IDs "xxx_1  ".

Comment (ROCO-freedom) [reply base_tasofro/th175/data/script/scene/pause.v1.14.nut.map:19]: But I found nut_strings have some strings appeared more than once.How about cleaning up duplicate strings?

Comment (brliron) [reply base_tasofro/th175/data/script/scene/pause.v1.14.nut.map:19]: I'll look at cleaning up duplicate strings later (unless you want to take care of that). For now, I'll merge this.

**Pull request merged**